fix_warnings.py

```
@@ -0,0 +1,27 @@
+import re
+
+file_path = 'crates/tokmd/src/commands/cockpit.rs'
+
+with open(file_path, 'r') as f:
```

Ruff (0.14.14)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)